src/sidebar/services/annotations-exporter.ts

```diff
@@ -26,7 +27,9 @@ export class AnnotationsExporter {
    */
   buildExportContent(now = new Date()): ExportContent {
     const profile = this._store.profile();
-    const annotations = this._store.allAnnotations();
+    const annotations = this._store
+      .allAnnotations()
+      .map(stripInternalProperties) as Annotation[];
```

Comment:
Here's the thing though: the returned objects from `stripInternalProperties` are not `Annotation`s even if you feed `Annotation`s in. If you look at the typing for `Annotation` in `types/api`, `$tag` is a required field both in `ClientAnnotationData` and `Annotation` itself.

The typing itself here is problematic, as the `Annotation` type sort of claims it's an "API" type, but it's polluted by client details. I'ma give this a think.

Reply:
I'm not sure if I fully understand the comment. Do you mean that the property `$tag` should still be kept in the export file?